wiki/Consciousness and Self-Regualtion in Mobile Communication.qmd

```diff
@@ -0,0 +1,18 @@
+---
+title: "J.B. Bayer et al. (2016). Consciousness and Self-Regualtion in Mobile Communication. *Human Communication Research 42 (2016) 71–97."
+subtitle: "BA New Media / Online Communication KF/BF A: Spring Term 2023"
+author: "Judith Kaß"
+categories:
+  - "Digital Media"
+
+**(K):** “By integrating Studies 1 and 2, we see how different forms of media cognition are linked and, at the same time, how they differ from each other in their relationships to overall self-regulation.“ (p. 89)
+**(A):** The paper mostly argues that the media use are a mix of automatic and immersive habits but peoples cognitive and individual tendencies while using media might differ. In this study self-regulation and everybody’s ability to self-regulate is a personal tendency. 

```

Comment:
Can you try to make an argument how these habits might relate to individuals' autonomy need experiences?

Reply:
Individuals' autonomy need experience can be influenced by their media use habits, self-regulation abilities, and personal tendencies. Strong self-regulation skills can empower individuals to exercise autonomy over their media use, while weaker self-regulation skills may lead to a diminished sense of autonomy. Additionally, individuals' varying needs for autonomy can shape their preferences for different types of media experiences. Understanding these dynamics can help individuals make conscious choices about their media consumption and cultivate a more fulfilling relationship with media.